### PR TITLE
Tutorial on weighted directed graph representation for polymers

### PR DESCRIPTION
**Pull request by TRY-ER**
## Description

Fix #(issue)

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [x] Documentations (modification for documents)

## Checklist

- [ ] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [ ] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [ ] Run `mypy -p deepchem` and check no errors
  - [ ] Run `flake8 <modified file> --count` and check no errors
  - [ ] Run `python -m doctest <modified file>` and check no errors
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New unit tests pass locally with my changes
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: examples/tutorials/Understanding_WDGraphs_for_polymers.ipynb [hunk 1]
```diff
@@ -0,0 +1,1289 @@
+{
```
Comment (shreyasvinaya) [inline examples/tutorials/Understanding_WDGraphs_for_polymers.ipynb:1]: <p>add subsections</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (shreyasvinaya) [inline examples/tutorials/Understanding_WDGraphs_for_polymers.ipynb:1]: <p>reduce the image size</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Comment (shreyasvinaya) [inline examples/tutorials/Understanding_WDGraphs_for_polymers.ipynb:1]: <p>elaborate the points</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Comment (TRY-ER) [reply examples/tutorials/Understanding_WDGraphs_for_polymers.ipynb]: done

Comment (TRY-ER) [reply examples/tutorials/Understanding_WDGraphs_for_polymers.ipynb]: done

Comment (TRY-ER) [reply examples/tutorials/Understanding_WDGraphs_for_polymers.ipynb]: done

**Comment by TRY-ER** [comment]
having some conflict. Raising an new one with clean edit.

**Pull request closed without merging**